src/proto.rs

```diff
@@ -69,10 +69,11 @@ impl WebSocketResponseMessage {
 }
 
 impl SyncMessage {
-    pub fn with_padding<R: Rng + CryptoRng>(csprng: &mut R) -> Self {
-        let random_size = csprng.gen_range(1..=512);
+    pub fn with_padding() -> Self {
+        let mut rng = thread_rng();
```

Comment:
Shouldn't we just provide the rng, like we do everywhere else in the API? I wouldn't depend on a specific Rng.

Reply:
Sure. Let me restore it